frontend/compositions/visualizations/map/Map.tsx

```diff
@@ -198,3 +202,11 @@ export default function Map() {
     <div></div>
   )
 }
+
+export const BodyComp = (bodyContent: string | any) => {
```

Comment:
I'm having trouble finding where this is used - seems a bit unnecessary to factor this out given its simplicity, and the div is probably unnecessary - could we potentially just put a paragraph with the body content wherever this is called?